influence_benchmark/RL/expert_iteration.py

```diff
@@ -36,12 +36,16 @@ def __init__(
         ), "num_gen_trajectories must be higher than (num_envs_per_device +1) * num_devices"
 
         if run_name is None:
-            self.run_name = datetime.now().strftime("%Y-%m-%d_%H-%M-%S")
+            self.run_name = env_args["env_name"] + str(datetime.now().strftime("%m-%d_%H-%M-%S"))
         else:
             self.run_name = run_name
         self.env_args = env_args
         self.training_args = training_args
+
+        self.training_args["output_dir"] = str(PROJECT_DATA / "models" / self.run_name)
```

Comment:
Maybe makes sense to put the models in the run directory? Maybe you have a reason for not doing that